mfs/dir.go

```diff
@@ -126,7 +126,7 @@ func (d *Directory) childNode(name string) (FSNode, error) {
 		ndir := NewDirectory(d.ctx, name, nd, d, d.dserv)
 		d.childDirs[name] = ndir
 		return ndir, nil
-	case ufspb.Data_File:
+	case ufspb.Data_File, ufspb.Data_Raw:
```

Comment:
Hey @whyrusleeping, doesn't this fix imply that the `truncate` option is changing the file type from `Data_File` to `Data_Raw`? Is this intended?